patches/minecraft/net/minecraft/entity/merchant/villager/VillagerEntity.java.patch

```diff
@@ -9,7 +9,15 @@
           if (this.func_70631_g_()) {
              this.func_213756_es();
              return ActionResultType.func_233537_a_(this.field_70170_p.field_72995_K);
-@@ -634,7 +634,8 @@
+@@ -489,6 +489,7 @@
+       }
+ 
+       this.field_70180_af.func_187227_b(field_213775_bC, p_213753_1_);
++      if(villagerdata.func_221130_b() != p_213753_1_.func_221130_b()) net.minecraftforge.event.ForgeEventFactory.onVillagerProfessionChange(this, villagerdata.func_221130_b());
```

Comment:
Spaces before (s
And a second line after the if for the event
Having things run off the side of the screen is a PITA to read

Reply:
My understanding of the PR guidelines was to keep things in patches to 1 line if humanly possible. But I've now split it to a 2nd line.

Reply:
we prefer not having *large* patches, but splitting on long lines helps readability a ton.

Reply:
Fair enough, makes sense